src/components/Form/Form.react.js

```diff
@@ -37,6 +37,7 @@ export type Props = {|
   +className?: string,
   +action?: string,
   +method?: string,
+  +autocomplete?: "on" | "off",
```

Comment:
If there is a `defaultProps` for `autocomplete` it should not be optional, what do you think?